content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -203,7 +203,7 @@ kube-dns   ClusterIP   10.0.0.10    <none>        53/UDP,53/TCP   8m
 
 If it isn't running, you can [enable it](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/kube-dns/README.md#how-do-i-configure-it).
 The rest of this section will assume you have a Service with a long lived IP
-(my-nginx), and a DNS server that has assigned a name to that IP (the CoreDNS
+(my-nginx), and a DNS server that has assigned a name to that IP (the Kube-dns
```

Comment:
In fact, this cluster addon is CoreDNS with service name still kube-dns.

Reply:
Well, in that case, I think this part of the documentation is very confusing.
<img width="1282" alt="Screen Shot 2019-11-14 at 1 00 54 PM" src="https://user-images.githubusercontent.com/4717349/68835783-d5575c00-06de-11ea-9710-0c1ac45f8609.png">

A short description about it is named as "kube-dns", but it is CoreDNS behind the scene can be added (till the time the service name is corrected).
Also the link on "enable it" is pointing towards kube-dns deployment only (not CoreDNS). So if someone doesn't have it enabled, they will end up enabling kube-dns instead.

Reply:
Maybe “the CoreDNS cluster add-on (application name `kube-dns`)”?

The guide to enabling kube-dns should be about making sure that CoreDNS is running.

Reply:
Kube-dns cluster add-on has a README in https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns/kube-dns

But CoreDNS cluster add-on doesn't have one in https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns/coredns

Is there any good link about how to enable CoreDNS on Kubernetes?

Reply:
Are these links help?
[https://github.com/coredns/deployment/tree/master/kubernetes](https://github.com/coredns/deployment/tree/master/kubernetes)
[https://kubernetes.io/docs/tasks/administer-cluster/coredns/](https://kubernetes.io/docs/tasks/administer-cluster/coredns/)

Reply:
Yes. These links look good. Let's give the user both. Updating the PR..